interfaces/Portalicious/src/app/pages/project-registrations/project-registrations.page.ts

```diff
@@ -73,45 +75,29 @@ export class ProjectRegistrationsPageComponent {
 
   project = injectQuery(this.projectApiService.getProject(this.projectId));
 
-  readonly canChangeStatus = computed(
-    () =>
-      (
-        status:
-          | RegistrationStatusEnum.declined
-          | RegistrationStatusEnum.deleted
-          | RegistrationStatusEnum.included
-          | RegistrationStatusEnum.paused
-          | RegistrationStatusEnum.validated,
-      ) => {
-        if (
-          status === RegistrationStatusEnum.validated &&
-          !this.project.data()?.validation
-        ) {
-          return false;
-        }
+  readonly canChangeStatus = computed(() => {
+    const project = this.project.data();
+    if (!project) {
+      return () => false;
+    }
+
+    return (
+      status:
+        | RegistrationStatusEnum.declined
```

Comment:
Would we benefit from creating a interface/enum which is this subset of registration statusses. As we use it  in multiple places? 
@aberonni